### ISSUE TITLE
Marketplace

### ISSUE THREAD

**Issue by arbiterFF** [enricoros/big-AGI#373]
**Why**
I want to access other people's personas

**Description**
Introduce the ability for folks to make their personas public on a marketplace so that third parties can query their data. For instance: I want to make an Allygn persona who is an expert at SF city politics.

**Comment by enricoros** [comment]
Good idea, will need differentiation with the GPTs marketplace. Appended to the roadmap.
